tests/github/fair_software/doesnotexist/test_repo_no_args.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+from requests_mock import Mocker
+from howfairis import Repo
+from howfairis.exceptions import GetDefaultBranchException
+from tests.contracts.repo import Contract
+
+
+def get_mocked_repo():
```

Comment:
I know I started this pattern :), but I was thinking of changing `get_mocked_repo` to `get_repo`. What do you think?

Reply:
Agree, the repo is not mocked, but the repo on the Internet is faked with the `mocker` fixture.

Reply:
I renamed in the files I wrote in this PR, can you create a issue/pr for the others?

Reply:
done, #228